js/modules/k6/grpc/client_test.go

```diff
@@ -1,16 +1,11 @@
-package grpc
+package grpc_test
```

Comment:
Out of curiosity, why are we moving them to a specific package? Do we have cycle dependencies or other issues? I'm not against it, but I've seen some ongoing discussions around gRPC and extensions and I have the feeling I lost something.

Reply:
I'd say it's just a better way of structuring the tests (best practice), basically test package becomes a user of the main package, which blocks access to the main package's internals, and as a result, we should write more robust tests, since even something internally in the package changed the tests continue to works since they rely on the public API